docs/src/Typeahead.doc.js

```diff
@@ -203,25 +216,100 @@ function TypeaheadExample() {
   return (
     <Flex gap={4}>
       <Typeahead
-        label="Select  your favorite shape"
+        label="Select your favorite shape"
         id="favorite-shape"
         noResultText="No Results"
         options={[{label:'square', value:'square'}, {label:'circle', value:'circle'}]}
         onSelect={p => ref.current.focus()}
         placeholder="Select a shape"
       />
-       <Typeahead
-        label="Select  your favorite color"
+      <Typeahead
+        label="Select your favorite color"
         id="favorite-color"
         noResultText="No Results"
         options={[{label:'red', value:'red'}, {label:'blue', value:'blue'}]}
         placeholder="Select a color"
         ref={ref}
-     />
+      />
     </Flex>
   );
 }`}
   />
 );
 
+card(
+  <Example
+    id="tagsExample"
```

Comment:
It seems that if you type something not included in the list of options, and then hit enter, it deletes everything you typed. I think instead it should leave what you typed, without adding it as a tag. Deleting everything typed could cause frustration is users simply made a typo and then have to re-type everything

Reply:
That's just the current behavior of the Typeahead component, unrelated to tags 😕

Reply:
Ahh good catch, ignore that then!

Reply:
Yeah I think that's maybe where a bigger amount of work needs to be done around adding tags and validation around tags in a typeahead. I'm assuming that's a decent chunk of work?

Reply:
Could you elaborate @ashleyseto? Are you thinking that if people try to add a tag in typeahead that's not one of the options that we warn/alert the user it's not an option? Or would we have an option to add tags that aren't included in the options list? (or both 😛 )

Reply:
Oh I mean that we could have a select from list of items but an option to 'add your own' item to a list if this was a category of items. I don't have a use case for this yet that I know of so it isn't important.

The original thinking with the typeahead was to clear invalid items to avoid error states. In our case now though, we do offer an error tag that we could use if there's feedback that clearing is an issue.

Reply:
Makes sense, we'll keep an eye out and see